src/Microsoft.AspNetCore.SignalR.Client.Core/ConnectionFactoryDelegate.cs

```diff
@@ -1,9 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using Microsoft.AspNetCore.Sockets.Client;
-
-namespace Microsoft.AspNetCore.SignalR.Client
-{
-    public delegate IConnection ConnectionFactoryDelegate();
```

Comment:
Out of curiosity - is it being removed because we just expose Func everywhere else?

Reply:
Yeap